wenv

```diff
@@ -314,14 +314,17 @@ OPTIONS
     shift $((OPTIND-1))
 
     [[ -z "$1" ]] && return 1
-    local wenv="$1"
 
     # TODO: generalize/clean this up
     if [[ $flag_d -eq 1 ]]; then
-        cat =(perl -pe "s|WENV_DIR=.*?$|WENV_DIR=\"`pwd`\"|" "$template") > "$WENV_CFG/wenvs/$wenv"
+        [[ -z "$2" ]] && echo "directory path and wenv name arguments are required with the -d flag" && return 1
+        local wenv_dir="$1"
```

Comment:
This should be set above the getopt call to default to `pwd`, then overwritten with `OPTARG` if `-d` is passed. See [this](https://github.com/dgrisham/wenv/blob/master/wenv#L615) for an example of how to use `OPTARG`.

Reply:
Doing it this way will remove the `if`/`else` here as well

Reply:
Good call. Thoughts on the `-n` operator over `! -z` wasn't sure if those would accomplish the same thing.